doc/nrf/ug_thread_commissioning.rst

```diff
@@ -80,6 +80,25 @@ Minimal Thread Device (MTD)
 
 For more information, see `Device Types on OpenThread portal`_.
 
+SED vs SSED activity
+--------------------
+
+.. thread_ssed_description_start
+
+A Thread Synchronized Sleepy End Device is synchronized with its parent router and uses the radio only at scheduled intervals, by using the Coordinated Sampled Listening (CSL) feature introduced in the Thread v1.2.
```

Comment:
```suggestion
A Thread Synchronized Sleepy End Device (SED) is synchronized with its parent router and uses the radio only at scheduled intervals, by using the Coordinated Sampled Listening (CSL) feature introduced in the Thread v1.2.
```
Since this is an include, I think it's better to have the acronyms defined inside it as well.

Should CSL link to the definition? No current ref tag on the header.

Reply:
Added ref to `Thread 1.2 Base Features`